### PR TITLE
Convert toolspipeline to cronjob

### PR DESCRIPTION
**Pull request by jeanmi151**
cf https://github.com/geonetwork/geonetwork-helm-charts/issues/12

### CODE REVIEW

File: gn-tools-pipelines/templates/gntoolspipelines-job.yaml
Comment (edevosc2c) [inline gn-tools-pipelines/templates/gntoolspipelines-job.yaml:1]: could you rename the file to gntoolspipelines-cronjob.yaml :)?

**Review by edevosc2c — CHANGES_REQUESTED** [review]

**Comment by edevosc2c** [comment]
If you want to create a nice cronjob helm chart you should look into this helm chart: https://github.com/camptocamp/charts-gs/tree/main/georchestra-backup-ldap

Right now in your helm chart you can't change many important things: schedule, successfulJobsHistoryLimit, restartPolicy

File: gn-tools-pipelines/templates/gntoolspipelines-job.yaml
Comment (jeanmi151) [reply gn-tools-pipelines/templates/gntoolspipelines-job.yaml:1]: still working on it no worries

**Comment by jeanmi151** [comment]
Seems to be ready
I deployed it somewhere and it works

**Comment by jeanmi151** [comment]
tested and working @edevosc2c if you have anything to add please

**Review by edevosc2c — APPROVED** [review]

**Pull request merged**